com.github.mimillieuh.thinkercad-desktop.metainfo.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>com.github.mimillieuh.thinkercad-desktop</id>
+  <name>ThinkerCAD Dekstop</name>
+  <summary>A simple ThinkerCAD wraper for desktop</summary>
+  <developer_name>MiMilleuh</developer_name>
+  <url type="homepage">https://github.com/MiMillieuh/ThinkerCAD-Desktop</url>
+  <url type="bugtracker">https://github.com/MiMillieuh/ThinkerCAD-Desktop/issues</url>
+  <metadata_license>GPL-3.0</metadata_license>
```

Comment:
this is not an acceptable **metadata** license. `CC0-1.0` is preferred, but other options can work.